app/workers/organizations/delete_worker.rb

```diff
@@ -0,0 +1,29 @@
+module Organizations
+  class DeleteWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    # user_id - the user who deletes the organization
+    def perform(organization_id, user_id)
```

Comment:
I wonder if we should rename this value to `operator_id` and store this somewhere.

We don't leave a trace of who deleted which organization. Should we use `Audit::Log` to store this info ?

Reply:
Thanks for your suggestions, @rhymes. I changed the variable name, as for the `Audit::Log`, I think it makes sense to implement it as a next step.

Reply:
I ended up adding an `AuditLog` :)